cdi/embedded/src/test/java/org/infinispan/cdi/embedded/test/distexec/DistributedExecutorCDITest.java

```diff
@@ -89,11 +83,7 @@ public void run() {
       }
    }
 
-   static class ImpliedInputCacheCallable implements Callable<Integer>, Serializable {
-
-
-      /** The serialVersionUID */
-      private static final long serialVersionUID = 5770069398989111268L;
+   static class ImpliedInputCacheCallable implements Callable<Integer>, Serializable, ExternalPojo {
```

Comment:
Adding `ExternalPojo` was crucial here. But I'm not sure why to be honest, @galderz any opinions?

I can see it's a consequence of this commit https://github.com/infinispan/infinispan/commit/82d268bdea0b2bb3bef4eed1a8bda4ceeaf91409. So I'm wondering what is the impact for the end user? If one creates a new `Callable` and submits it to our `DistributedExecutorService`, does it mean that the submitted class needs to implement `ExternalPojo` as well?